mock/src/lib.rs

```diff
@@ -21,8 +21,14 @@ lazy_static! {
         address!("0x00000000000000000000000000000000c014ba5e");
     /// Mock gasprice value
     pub static ref MOCK_GASPRICE: Word = Word::from(1u8);
+    /// Mock BASEFEE value
+    pub static ref MOCK_BASEFEE: Word = Word::zero();
```

Comment:
I'd probably use a value that is different from 0. Otherwise some tests might pass when shouldn't. But we should keep the MockBlock::default() with 0 still.

Reply:
> I'd probably use a value that is different from 0. Otherwise some tests might pass when shouldn't. But we should keep the MockBlock::default() with 0 still.

I've tried to make `MOCK_BASEFEE: Word = Word::from(1u64)`, but when I run this test. It shows error, like below.
<img width="442" alt="image" src="https://user-images.githubusercontent.com/108982045/205432280-708f73bd-a68e-45a7-af78-8ff12ab0b155.png">


Seems like, if we did this, MOCK_BASEFEE will not be used in the unit test for BASEFEE.

Or, let MOCK_BASEFEE andbase_fee_per_gas in  MockBlock::default be equal.

Reply:
Is it possible that there are tests that need to update the BaseFee to the new value used??

Reply:
For now, seems like no one wanna update it.

Reply:
Feel free to leave it to zero then. I'll take a deeper look during next weeks.